api-client/src/maps/index.ts

```diff
@@ -11,6 +11,11 @@ interface MapViewViewport {
   zoom: number
 }
 
+interface MapIcons {
```

Comment:
new possibilities of icons gonna be added when needed

Reply:
i think lateron that we need a Map with  category->icon-name (a svg icon set)
This is just a temporary solution

Reply:
i replaced it by sprite icons